web/src/cordova.js

```diff
@@ -28,25 +28,56 @@ export const cordovaApp = {
   onDeviceReady() {
     // Resize if we're on iOS
     if (cordova.platformId === 'ios') {
-      // TODO(olc): What about Xr and Xs?
-      const extraPadding = (device.model === 'iPhone10,3' || device.model === 'iPhone10,6')
-        ? 30
-        : 20;
-      select('#header')
-        .style('padding-top', `${extraPadding}px`);
-      select('#mobile-header')
-        .style('padding-top', `${extraPadding}px`);
-
-      select('.controls-container')
-        .style('margin-top', `${extraPadding}px`);
-
-      select('.flash-message .inner')
-        .style('padding-top', `${extraPadding}px`);
-
-      select('.mapboxgl-zoom-controls')
-        .style('transform', `translate(0,${extraPadding}px)`);
-      select('.layer-buttons-container')
-        .style('transform', `translate(0,${extraPadding}px)`);
+
+      const styles = function (top, bottom) {
+        return `
+        /* TODO: this selects nothing, header on iPad still overlaps with the status bar */
+        #header {
+          padding-top: ${top};
+        }
+
+        #mobile-header {
+          padding-top: ${top};
+        }
+
+        @include respond-to('small') {
```

Comment:
I haven't seen the `include respond-to` syntax before. Is this valid?

Reply:
Removing `@include respond-to('small') {` solves the issue where all the CSS rules below are not applied.

Reply:
There's 30 occurrences of that in the files (well, 29 without this one), so yeah, it works. It's needed because padding does not need to be applied when there's a header (on iPad). 
It worked on my machine (c) but perhaps it's missing an import? I'm checking locally as well :)

Reply:
It is a scss/sass construct that doesn't work in plain CSS (which we are using here). The occurences are in .scss files that are preprocessed.

Reply:
right 🤦 then it's an easy fix, just replacing that with a media selector. I can do it, but later - otherwise go ahead ☺️

Reply:
I tested on a web version a lot because I couldn't stand waiting 3 min of rebuild after every change. and there it obviously worked 😅

Reply:
I'll add the media query to https://github.com/tmrowco/electricitymap-contrib/pull/3132 :)

Reply:
thanks 👍

Reply:
> waiting 3 min of rebuild after every change. and there it obviously worked 😅

I hope to get time looking into a better local development flow – the current one is awful. 😅

Reply:
would be an awesome improvement 😄